docs/dev/analysis/tbl-cell-borders.rst

```diff
@@ -0,0 +1,92 @@
+
+Table Cell Borders
+------------------
+
+In PowerPoint, a table cell can have borders on all 4 sides, plus two diagonal lines (bottom left to top right and
+top left to bottom right) for a total of 6 possible borders, each specified by a child element of the table cell
+properties element.
+
+Each border is further customized by a set of attributes and child elements common to all borders (`LineFormat`).
+
+In order to provide `LineFormat` functionality, each border `_ln` object is wrapped in a `_TableBorderLine`,
+which implements the `get_or_add_ln` method, and can be then wrapped by `LineFormat`.
+
+Protocol
+--------
+
+**Starting with a default cell** A new cell has no borders or style.
+New borders can be created by accessing the borders property::
+
+    >>> table = shapes.add_table(rows, columns, left, top, width, height).table
+    >>> cell = table.rows[0].cells[0]
+    >>> cell
+    <pptx.shapes.table._Cell object at 0x...>
+    >>> cell.borders
+    <pptx.shapes.table._TableBorders object at 0x...>
+
+**Create a border** A cell border is created by applying line width and color for its `LineFormat`.
+This can be done with any one of the 6 cell borders::
+
+    >>> cell.borders.top.width = Pt(1)
+    >>> cell.borders.top.color.rgb = RGBColor(0, 0, 0)
+    >>> cell.borders.top._ln.xml
+    <a:lnT w="12700">
```

Comment:
Just for consistency, let's put the XML in a separate 'Specimen XML' section. I'd start with one that represents a default cell as generated by PowerPoint. Then a second one for a cell that has a border. Add another if there's anything interesting about a cell with multiple borders or whatever. You can find examples in other analysis pages. I'd start at the `a:tc` level as you have here, but you can truncate the elements we don't need to dig into, like `<a:txBody .../>`.